packages/backend/src/utils/npm.ts

```diff
@@ -135,8 +142,15 @@ class Command {
     }
   }
 
-  public getGlobalNodeModulesPath(): string {
-    return this.globalNodeModulesPath;
+  private async getGlobalPath(): Promise<string> {
+    const globalNodeModulesPath = await this.getGlobalNodeModulesPath();
+    const nmLength = path.join(path.sep, "node_modules").length;
+    return globalNodeModulesPath.substring(0, globalNodeModulesPath.length - nmLength);
```

Comment:
risky method. better try :
  _.join(_.without(_.split(globalNodeModulesPath, path.sep), 'node_modules'), path.sep) 
up to you

Reply:
Done